src/index.html

```diff
@@ -7,12 +7,72 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__top">
+        <a
+          href="#"
+          class="card__image"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__img"
+          />
+        </a>
+      </div>
+
+      <div class="card__bottom">
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <span class="card__code">Product code: 195434</span>
```

Comment:
```suggestion
        <h3 class="card__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h3>
        <span class="card__code">Product code: 195434</span>
```

Reply:
@anastasiiavorobiova  Why should i change h1 to h3??? No h1 or h2 before, could you explain, please?